<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>905</width>
      <height>584</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>2</number>
     </property>
     <item row="0" column="13">
      <widget class="QSpinBox" name="numberOfParticleSpinBox">
       <property name="minimum">
        <number>21</number>
       </property>
       <property name="maximum">
        <number>900</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of Particle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of Iterations</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QDoubleSpinBox" name="tSpinBox">
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="13">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QDoubleSpinBox" name="alphaSpinBox">
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>alpha</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="11">
      <widget class="QLabel" name="status_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_t">
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="12">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>512</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>tSpinBox</tabstop>
  <tabstop>numberOfParticleSpinBox</tabstop>
  <tabstop>pauseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
